examples/cancellable_server.rs

```diff
@@ -0,0 +1,36 @@
+use open62541::ServerBuilder;
+use std::time::Duration;
+use tokio_util::sync::CancellationToken;
+use tokio_util::task::TaskTracker;
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    env_logger::init();
+
+    let task_tracker = TaskTracker::new();
+    let task_tracker_clone = TaskTracker::new();
```

Comment:
Should this line be the following?
`let task_tracker_clone = task_tracker.clone():`

Reply:
We actually don't need any async code and `tokio-util` here, see https://github.com/ndchandar/open62541/pull/1.

Reply:
Btw, thanks for chiming in! I didn't expect that this project is already gaining attention :sweat_smile: